internal/lookout/ui/src/components/jobs/ReprioritizeJobs.tsx

```diff
@@ -0,0 +1,110 @@
+import React, { Ref } from "react"
+
+import {
+  Paper,
+  Table,
+  TableBody,
+  TableCell,
+  TableContainer,
+  TableHead,
+  TableRow,
+  Theme,
+  createStyles,
+  TextField,
+} from "@material-ui/core"
+import { makeStyles } from "@material-ui/core/styles"
+
+import { Job } from "../../services/JobService"
+import LoadingButton from "./LoadingButton"
+
+const useStyles = makeStyles((theme: Theme) =>
+  createStyles({
+    paper: {
+      backgroundColor: theme.palette.background.paper,
+      boxShadow: theme.shadows[5],
+      padding: theme.spacing(2, 4, 3),
+      outline: "none",
+      borderRadius: "0.66em",
+      maxHeight: "80%",
+      maxWidth: "75%",
+      display: "flex",
+      flexDirection: "column",
+    },
+    container: {
+      overflow: "auto",
+      margin: theme.spacing(0, 0, 1),
+    },
+    button: {
+      display: "flex",
+      alignItems: "center",
+      justifyContent: "center",
+      marginTop: "1em",
+    },
+  }),
+)
+
+type ReprioritizeJobsProps = {
+  jobsToReprioritize: Job[]
+  isLoading: boolean
+  isValid: boolean
+  onReprioritizeJobs: () => void
+  onPriorityChange: (e: any) => void
+}
+
+const ReprioritizeJobs = React.forwardRef((props: ReprioritizeJobsProps, ref: Ref<any>) => {
+  const classes = useStyles()
+
+  return (
+    <div ref={ref} className={classes.paper}>
+      <h2 id="reprioritize-jobs-modal-title" className="reprioritize-jobs-modal-title">
+        Reprioritize jobs
+      </h2>
+      <p id="reprioritize-jobs-modal-description" className="reprioritize-jobs-modal-description">
+        The following jobs will be reprioritized:
+      </p>
+      <TableContainer component={Paper} className={classes.container}>
+        <Table stickyHeader>
+          <TableHead>
+            <TableRow>
+              <TableCell>Id</TableCell>
+              <TableCell>Job Set</TableCell>
+              <TableCell>State</TableCell>
+              <TableCell>Current Priority</TableCell>
+              <TableCell>Submission Time</TableCell>
+            </TableRow>
+          </TableHead>
+          <TableBody>
+            {props.jobsToReprioritize.map((job) => (
+              <TableRow key={job.jobId}>
+                <TableCell>{job.jobId}</TableCell>
+                <TableCell>{job.jobSet}</TableCell>
+                <TableCell>{job.jobState}</TableCell>
+                <TableCell>{job.priority}</TableCell>
+                <TableCell>{job.submissionTime}</TableCell>
+              </TableRow>
+            ))}
+          </TableBody>
+        </Table>
+      </TableContainer>
+      <TextField
```

Comment:
![image](https://user-images.githubusercontent.com/14963658/122237616-5f693700-ceb7-11eb-8ab8-a9f3ab2f4da4.png)
Might be worth making this TextField a bit shorter